packages/replay/src/main.ts

```diff
@@ -1,19 +1,12 @@
 import dbg from "debug";
 import fs from "fs";
 import path from "path";
-import {
-  initConnection,
-  connectionCreateRecording,
-  connectionProcessRecording,
-  connectionWaitForProcessed,
-  connectionUploadRecording,
-  connectionUploadSourcemap,
-  connectionReportCrash,
-  closeConnection,
-  setRecordingMetadata,
-  buildRecordingMetadata,
-  connectionUploadOriginalSource,
-} from "./upload";
+
+// requiring v4 explicitly because it's the last version with commonjs support.
+// Should be upgraded to lhe latest when converting this code to es modules.
```

Comment:
`lhe` => `the`

Reply:
![](https://media.tenor.com/8Ys5O0Y00m8AAAAM/angry-mad.gif)